xds/internal/xdsclient/xdslbregistry/xdslbregistry_test.go

```diff
@@ -46,7 +46,8 @@ import (
 	"google.golang.org/grpc/serviceconfig"
 	"google.golang.org/grpc/xds/internal/balancer/ringhash"
 	"google.golang.org/grpc/xds/internal/balancer/wrrlocality"
-	"google.golang.org/grpc/xds/internal/xdslbregistry"
+	"google.golang.org/grpc/xds/internal/xdsclient/xdslbregistry"
+	_ "google.golang.org/grpc/xds/internal/xdsclient/xdslbregistry/converter" // To register converters.
```

Comment:
This needs to go into `google.golang.org/grpc/xds` itself, I think.  This package can import `xds` instead.

Reply:
Pls also move the pb imports above into their own group at the bottom.

Reply:
Done.